src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -257,7 +257,7 @@ def exit(self):
         ]
         for container_id in active_container_ids:
             try:
-                subprocess.Popen(f"docker kill {container_id}", shell=True)
+                subprocess.Popen(f"docker stop {container_id}", shell=True)
             except Exception:
                 pass
```

Comment:
This entire `exit` method definition can be replaced with Python Docker API calls (starting from line 247), which is going to be much cleaner and less error prone than parsing shell output like this. I think this PR may be a good opportunity to review this `exit` method (or in a separate PR when time permits)

Reply:
@raph-luc It's update now. We are using just `self.server.kill()` with help of Docker SDK. Thank you.